tests/test_streaming.py

```diff
@@ -59,6 +59,203 @@ def test_dataloader_epoch_size_no_streams(local_remote_dir: Tuple[str,
             assert samples_seen == epoch_size
 
 
+@pytest.mark.parametrize('batch_size', [4, 7])
+@pytest.mark.parametrize('seed', [2222])
+@pytest.mark.parametrize('shuffle', [True])
+@pytest.mark.parametrize('drop_last', [True])
+@pytest.mark.parametrize('num_workers', [1])
```

Comment:
should we put `num_workers` as `4` or `8` instead of `1` ?

Reply:
yep!